RedHat-UX/red-hat-design-system
# fix(footer): add explicit typography tokens to headers

## What I did

1. Added line-height and font-family tokens to all headers in the lightdom and shadowdom.
2. Added !important to font related properties in the shadowroot.
3. Removed unneeded wildcard `*` for our `:is()` and `:not` selectors.

## Testing Instructions

1. Go to https://deploy-preview-811--red-hat-design-system.netlify.app/elements/footer/demo/
4. Inspect the `<h3 slot="links">Communicate</h3>` heading.
5. It should contain the following css

```css
*:is(h1, h2, h3, h4, h5, h6) {
    font-family: var(--rh-font-family-heading, RedHatDisplay, "Red Hat Display", "Noto Sans Arabic", "Noto Sans Hebrew", "Noto Sans JP", "Noto Sans KR", "Noto Sans Malayalam", "Noto Sans SC", "Noto Sans TC", "Noto Sans Thai", Helvetica, Arial, sans-serif);
    line-height: var(--rh-line-height-heading, 1.3);
}
```
